lib/models/Session.js

```diff
@@ -11,9 +11,13 @@ class Session {
     this.respondedAt = respondedAt
     this.respondedByPhoneNumber = respondedByPhoneNumber
 
-    // this is temporary: change these two lines to `this.device = device` when renaming buttons and locations table to devices table in both buttons and sensor
+    // remove these two assignments when both BraveButtons and BraveSensor use this.device
     this.button = device
     this.location = device
+
+    this.device = device
```

Comment:
Oh cool. Looks like this will let you gradually change over references to `device` in the Buttons and Sensors codebases!

With this in, do you think the `sessionFactory` and `sessionDbFactory` could also move into `brave-alert-lib`? No need to do it now, I'm just planting the idea.

Reply:
I've decided to not move the factories to brave-alert-lib as they contain button-specific fake data, and sensor-specific fake data. To move these functions into brave-alert-lib would require modifying a lot of their usage. I'm expecting that to be absorbed into the work of merging the servers.